src/api/comments.js

```diff
@@ -0,0 +1,29 @@
+import { BASE_URL } from './api';
+
+export const getPostComments = async(postId) => {
+  const response = await fetch(`${BASE_URL}/comments`);
+  const result = await response.json();
+
+  return result.data.filter(comment => comment.postId === postId);
+};
+
+export const addPostComments = (newComment) => {
```

Comment:
```suggestion
export const addPostComment = (newComment) => {
```
you're adding single comment